src/TransactionController.ts

```diff
@@ -129,6 +138,48 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
 		});
 	}
 
+	/**
+	 * Normalizes the transaction information from etherscan
+	 * to be compatible with the TransactionMeta interface
+	 *
+	 * @param txMeta - Object containing the transaction information
+	 * @param currentNetworkID - string representing the current network id
+	 * @returns - TransactionMeta
+	 */
+	private normalizeTxFromEtherscan(txMeta: any, currentNetworkID: string): TransactionMeta {
+		const ts = parseInt(txMeta.timeStamp, 10) * 1000;
+		/* istanbul ignore next */
+		const status = txMeta.isError === '0' ? 'confirmed' : 'failed';
+		return {
+			blockNumber: txMeta.blockNumber,
+			id: random({ msecs: ts }),
+			networkID: currentNetworkID,
+			rawTransaction: '',
+			status,
+			time: ts,
+			transaction: {
+				from: txMeta.from,
+				gas: BNToHex(new BN(txMeta.gas)),
```

Comment:
It looks like we go from string -> BN -> hex. How is `txMeta.gas` originally expressed? Is it not in hex already? Same question for the other values below.

Reply:
It comes back as a string in gwei `"21000"`